src/_includes/effective-dart-banner.html

```diff
@@ -1,9 +1,9 @@
 <div class="banner">
-    <p class="banner__text">
-	<em>Effective Dart</em> has not yet been updated to Dart 2
-	and has some <a target="_blank" 
-        href="https://github.com/dart-lang/site-www/labels/EffectiveDart">known issues.</a>
-        If a guideline appears to be missing or doesn't make sense,
-	please file a bug.
-    </p>
+  <p class="banner__text">
+    <em>Effective Dart</em> is being updated to reflect
+    current practices and Dart 2.
+    If you notice a problem that isn't a <a target="_blank" 
+    href="https://github.com/dart-lang/site-www/labels/EffectiveDart">known issue,</a>
+    please report a bug.
```

Comment:
I prefer "issue" to "bug".

Reply:
I used "bug" because I wanted a reference to the bug icon at the upper right :)